packages/block-library/src/classic/edit.js

```diff
@@ -198,6 +202,9 @@ export default class ClassicEdit extends Component {
 				key="editor"
 				id={ `editor-${ clientId }` }
 				className="wp-block-freeform block-library-rich-text__tinymce"
+				contentEditable="true"
+				dangerouslySetInnerHTML={ { __html: this.props.attributes.content } }
```

Comment:
Why are we using `dangerouslySetInnerHTML` here? I don't see the content we're loading in here being sanitised anywhere here so I'm not sure it's safe to load. Even if it's safe upstream I would want to see it sanitised in this file for clarity's sake and I'd want to see tests making sure there is no potential for code injection here.